### PR TITLE
Add MinVerTagPrefix for SQL and gRPC instrumentation

### PR DESCRIPTION
**Pull request by alanwest**
In prep for release #5322.

SQL and gRPC instrumentation has not yet received a release that aligns major/minor version with the 1.7 API/SDK release.

### CODE REVIEW

**Review by CodeBlanch — APPROVED** [review]
LGTM

**Comment by utpilla** [comment]
@alanwest We could probably use a dedicated tag for Shims.OpenTracing as well (just for consistency)?

Would you want to update the RELEASING [doc](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/build/RELEASING.md)with these changes?

**Review by utpilla — APPROVED** [review]

File: src/OpenTelemetry.Instrumentation.GrpcNetClient/OpenTelemetry.Instrumentation.GrpcNetClient.csproj [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <TargetFrameworks>$(TargetFrameworksForGrpcNetClientInstrumentation)</TargetFrameworks>
     <Description>gRPC for .NET client instrumentation for OpenTelemetry .NET</Description>
     <PackageTags>$(PackageTags);distributed-tracing</PackageTags>
+    <MinVerTagPrefix>Instrumentation.Grpc-</MinVerTagPrefix>
```
Comment (utpilla) [inline src/OpenTelemetry.Instrumentation.GrpcNetClient/OpenTelemetry.Instrumentation.GrpcNetClient.csproj:7]: Match it with the project name?

```suggestion
    <MinVerTagPrefix>Instrumentation.GrpcNetClient-</MinVerTagPrefix>
```

**Comment by alanwest** [comment]
> Would you want to update the RELEASING [doc](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/build/RELEASING.md)with these changes?

Yep, sounds good. I'm going to first focus on getting the release out, but I will follow up with updating our release instructions.

**Pull request merged**